examples/map_transformations/azimuthal_equidistant_projection.py

```diff
@@ -0,0 +1,92 @@
+"""
+===============================================
+Azimuthal Equidistant Projection with Reproject
+===============================================
+
+In this example, we will reproject an AIA image into an
+`azimuthal equidistant projection <https://en.wikipedia.org/wiki/Azimuthal_equidistant_projection>`__.
```

Comment:
```suggestion
=====================================================
Reprojecting to a Map Projection with a Custom Origin
=====================================================

In this example, we show how to reproject a map to a map projection with a
custom origin.  Here, we choose the target map projection to be the
`azimuthal equidistant projection <https://en.wikipedia.org/wiki/Azimuthal_equidistant_projection>`__,
also known as the Postel projection, which has useful properties relative to a
specified origin of the projection.  If a different map projection is desired,
modifying this example is straightforward.
```
As I said, I want this example to be more general.  Also, you never actually explained that the azimuthal equidistant projection is the same thing as the Postel projection.